index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```

Comment:
- [ ] Kindly make sure you add the **stylesheet link** in your `head` tag to make the page be styled i.e your page can not be styled because the **stylesheet link** is not in the `head`.